automated_exports_test.go

```diff
@@ -0,0 +1,93 @@
+package recurly_test
+
+import (
+	"bytes"
+	"context"
+	"encoding/xml"
+	"log"
+	"net/http"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/aaronchooo/recurly"
```

Comment:
The fork thing is tripping me up.  I want to keep consistent with naming e.g. `recurly.AutomatedExport`, but I need to import from my forked repository since the main repo doesn't have `AutomatedExport` yet

Reply:
@aachou The easiest way I've found is to create a fork and add it as a remote to this repo locally. Then you can work on your fork inside of `blacklightcms/recurly` in your `GOPATH` and avoid dealing with import paths all together.

https://github.com/blacklightcms/recurly/pull/124#issuecomment-514352404